CryptoScan.Web/Main/Common/Http.cs

```diff
@@ -32,7 +32,7 @@ public async Task<ActionResult<T>> Get<T>(string url, string? notFoundError = nu
     }
     catch (Exception ex)
     {
-      return new BadRequestObjectResult($"{badRequestError}, error: {ex.Message}");
+      return new BadRequestObjectResult($"{badRequestError}, \nurl: {url}, \nerror: {ex.Message}");
```

Comment:
I think it's a good place to use the shiny new "raw string literals" feature :)
https://devblogs.microsoft.com/dotnet/csharp-11-preview-updates/

Reply:
I tried to update to the preview version of VS and it erased all my settings, so I rolled back to the latest stable version. Won't fix.